.ci.yaml

```diff
@@ -27,29 +28,32 @@ targets:
       - packages/flutter_tools/**
       - bin/**
 
-  - name: Linux customer_testing
-    builder: Linux customer_testing
+  - name: Linux build_tests_1_2
     properties:
       tags: >
-        ["framework","hostonly"]
+        ["framework","hostonly","shard"]
```

Comment:
what's the "shard" tag?

Reply:
I see, this was already on build_tests_1_2

Reply:
The diff on this PR is probably useless. I can upload the yaml lsc script, but it was just a sort() and remove the builder line.

https://paste.googleplex.com/4565275618836480?raw

Reply:
yeah, i trust the diff is just removing builder and sort. lgtm.